res/database/Default/user/scr0llbaer.xml

```diff
@@ -467,30 +510,121 @@ Format: Filmed lectures. Laid-back and to lay down.</en>
 				<modifier name="topicality::age" value="5" />
 				<modifier name="topicality::refresh" value="2" />
 			</programme_data_modifiers>
-		</scripttemplate-->
+		</scripttemplate>
 
-		<!--scripttemplate product="3" licence_type="1" guid="9f1cacc4-aa56-4f4a-9a9f-8c21c4bd3a7d" creator="9551" comment="Perhaps some reciprocal effect with the corresponding news genre can be implemented? TODO: I am not happy with the titles and descriptions of these.">
+		<scripttemplate product="3" licence_type="1" guid="56bcb1ff-8a5d-4d91-b3f7-ad6958b3855b" creator="9551">
+			<title>
+				<de>%recreational%%athletes% aus %randomcity% und anderswo</de>
+				<en>%recreational% %athletes% from %randomcity% and elsewhere</en>
+			</title>
+			<description>
+				<de>Aktuelle News und Berichterstattungen aus den Bunt-, Amateur- und Kreisligen der Nation. Fußball, Eishockey, Basketball, Football, Tennis, Tischtennis, Leichtathletik, Kirmesboxen, Kartsport und Formel 3 &amp; 4, und vieles mehr.</de>
+				<en>Latest news and coverage from the nation's hobby, amateur and county leagues. Soccer, ice hockey, basketball, football, tennis, athletics, fun fair boxing, karting and Formula 3 &amp; 4, and much more.</en>
+			</description>
+
+			<variables>
+				<recreational>
+					<de>Freizeit|Feierabend|Hobby|Amateur</de>
+					<en>Recreational|Leisure|Hobby|Amateur</en>
+				</recreational>
+				<athletes>
+					<de>athleten|kicker und co.|-Sportfreunde</de>
+					<en>Athletes|Footballers and co.|Sports Friends</en>
+				</athletes>
+				<randomcity>
+					<de>%STATIONMAP:RANDOMCITY%</de>
```

Comment:
remove one language entry, so the cities will be the same in all languages.
You can also add some custom names here `<de>Chemnitz|Leipzig|${STATIONMAP:RANDOMCITY}</de>`

Reply:
Just to clarify: only keep "<de>%STATIONMAP:RANDOMCITY%</de>". The engine will use this for all other localisations then. If you add multiple language tags here, each localization will get their own "replacement" and thus cities would be different if you change the language ingame.

Reply:
Yes. My Example was not good. (Add other city names only if they are the same in all languages. The canonical approach would be a city-Variabel `<city>Chemnitz|Leipzig|${randomcity}</city>`. Sorry for the confusion.)

Reply:
@nittka  - you might need to ensure that "non-locale-key"-Variables are threated correctly (direct values should go into the default locale key)

Reply:
removed the `en` entry.